doc/api/buffer.md

```diff
@@ -1977,7 +1977,9 @@ offset and cropped by the `start` and `end` indices.
 
 This is the same behavior as `buf.subarray()`.
 
-Note that this method is not compatible with the `Uint8Array#slice()`, which is a superclass of `Buffer`. If a copy of slice is needed, use `Uint8Array.prototype.slice` directly.
+Note that this method is not compatible with the `Uint8Array.prototype.slice()`,
```

Comment:
Feel free to ignore, but if you agree that it improves the text, here's my suggestion:

```suggestion
This method is not compatible with the `Uint8Array.prototype.slice()`,
```

Reply:
Thanks!